imagetest/testworkflow.go

```diff
@@ -390,6 +397,9 @@ func RunTests(ctx context.Context, testWorkflows []*TestWorkflow, outPath, proje
 	outFile.Write(bytes)
 	outFile.Write([]byte{'\n'})
 
+	if suites.Errors != 0 || suites.Failures != 0 {
+		return bytes, fmt.Errorf("test suite has error or failure")
```

Comment:
it is not a failure of `RunTests` if one of the tests has a failure status. let's make this function return the junit object rather than []byte, then caller can decide how to handle tests with errors